packages/db/test/cli/gen-migration.test.mjs

```diff
@@ -0,0 +1,45 @@
+import { once } from 'events'
+import { execa } from 'execa'
+import fs from 'fs'
+import split from 'split2'
+import { test } from 'tap'
+import { cliPath, getFixturesConfigFileLocation } from './helper.mjs'
+
+test('generates next file correctly', async ({ equal, match }) => {
+  const configFilePath = getFixturesConfigFileLocation('auto-apply.json')
+  const buffer = await fs.promises.readFile(configFilePath)
+  const config = JSON.parse(buffer.toString())
+
+  const migrationsBefore = await fs.promises.readdir(getFixturesConfigFileLocation(config.migrations.dir))
+  await execa('node', [cliPath, 'generate', 'migration', '-c', configFilePath])
+  const migrationsAfter = await fs.promises.readdir(getFixturesConfigFileLocation(config.migrations.dir))
+
+  const newMigrations = migrationsAfter.filter(migrationFile => !migrationsBefore.includes(migrationFile))
+  equal(newMigrations.length, 2)
+  match(newMigrations[0], '.do.sql')
+  match(newMigrations[1], '.undo.sql')
+
+  await Promise.all(newMigrations.map(migration => fs.promises.unlink(getFixturesConfigFileLocation(`${config.migrations.dir}/${migration}`))))
```

Comment:
Please use tmp dir instead.

Reply:
In the latest commit I made the changes. Can you verify this is what you had in mind ?

Reply:
Example of tmp dir usage:
https://github.com/platformatic/platformatic/blob/097a1486429a7bd04decc44bf6f3f1d6ca66fad5/packages/db/test/cli/init.test.mjs#L24

Reply:
I avoided the `tmpdir` because I need to generate a `platformatic.db.json` file for the test because the name of the directory is dynamic and the migrations path from the config is static.

So should I use `tmpdir` and generate a `platformatic.db.json` for this test ?

Reply:
You can copy your env to the tmp dir and execute it there. But it doesn't take much time so you can leave it like that.

Reply:
Thanks, appreciated.